### PR TITLE
Rm clamav related coding

### PR DESCRIPTION
**Pull request by zkdev**
Malware scan capabilities (based on ClamAV) were migrated to OCM-Gear extension.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Review by ccwienk — APPROVED** [review]
<3

**Review by 8R0WNI3 — APPROVED** [review]

File: concourse/model/traits/filter.py [hunk 1]
```diff
@@ -1,155 +0,0 @@
-# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```
Comment (8R0WNI3) [inline concourse/model/traits/filter.py:1]: This module is still used in delivery-service

Comment (ccwienk) [reply concourse/model/traits/filter.py:1]: @8R0WNI3 : it _is_? in that case, I suggest we remove usage asap. Code from concourse package is typically (exception: concourse/client.py) not intended for reuse outside of pipeline-template

**Review by ccwienk — CHANGES_REQUESTED** [review]
.

**Pull request merged**